pypeit/spectrographs/keck_lris.py

```diff
@@ -180,6 +190,11 @@ def compound_meta(self, headarr, meta_key):
                     return ('off' if plamps[len(old_lamp_names)-1] == '0' else 'on')
                 else:  # Lamp didn't exist.  Set to None
                     return 'None'
+        elif 'amp' in meta_key:
+            if self.name == 'keck_lris_red_mark4':
+                return headarr[0]['TAPLINES']
+            else:
+                return headarr[0]['NUMAMPS']
```

Comment:
Should this just be in `init_meta` of  class `KeckLRISRMark4Spectrograph`?

Reply:
yes, it should!  

moved.